### PR TITLE
add title field in SEO form by default

### PR DESCRIPTION
**Pull request by flaviaF**

### CODE REVIEW

File: MetaData/MetaDataResolver.php [hunk 1]
```diff
@@ -184,7 +184,14 @@ private function parseDefinition(array $definition, MetaData $metadata, Page $pa
             }
             $scheme = ('' === $metadata->getAttribute($attrname)) ? reset($attrvalue) : $metadata->getAttribute($attrname);
 
-            if (array_key_exists('default', $attrvalue)) {
+	    if (
+                    null !== $page
+                    && null !== $page->getTitle()
+                    && $metadata->getName() === 'title'
+		    && array_key_exists('default', $attrvalue)
+            ) {
```
Comment (eric-chau) [inline MetaData/MetaDataResolver.php]: Please take care about indentation and coding style =) Also you should test if $metadata is 'title' first so it can avoid you to do the check about `` null !== $page && null !== $page->getTitle()`` on every loop.

```php
if (
    'title' === $metadata->getName()
    && null !== $page
    && null !== $page->getTitle()
    && array_key_exists('default', $attrvalue)
) {
    $scheme = $page->getTitle();
}
```

**Comment by crouillon** [comment]
:+1: Thank you @flaviaF

**Comment by eric-chau** [comment]
thank you @flaviaF
